frontend/src/core/user/components/UserNotification.js

```diff
@@ -53,18 +56,34 @@ export default class UserNotification extends React.Component<Props, State> {
             >
                 <div className='item-content'>
                     <span className='actor'>
-                        <a href={notification.actor.url}>
-                            {notification.actor.anchor}
-                        </a>
+                        {is_comment ? (
```

Comment:
I think this would be cleaner with only one, high-level `is_comment` check. Even though we'd be repeating slightly more code, I find it hard to follow it with three `is_comment` checks. And we should do the same in the Django templates.